CMakeLists.txt

```diff
@@ -37,6 +35,12 @@ if(NOT ITK_INSTALL_INCLUDE_DIR)
 endif()
 string(REGEX REPLACE "^/" "" ITK_INSTALL_INCLUDE_DIR_CM24 "${ITK_INSTALL_INCLUDE_DIR}")
 
+# ====
+project( ITKMetaIO
+  VERSION "0.0.1"
+  DESCRIPTION "MetaImage is the text-based tagged file format for medical images that resulted. We have now extended that file format to support a variety of objects that occur in medicine such a tubes (for vessels, needles, etc.), blobs (for arbitrary shaped objects), cubes, spheres, etc. The complete library is known at MetaIO."
```

Comment:
"for medical images that resulted" -> sentence truncated

Reply:
It was copied and pasted from the Github description on the github site.

Reply:
@aylward do you have an updated description? To go both here and into GitHub website.